dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -105,9 +112,19 @@
  * @author dyuen
  */
 public abstract class AbstractEntryClient<T> {
+    private static final long LIST_RUNS_PAGE_SIZE = 60L;
+
+    private static final String AUTHORIZATION = "Authorization";
```

Comment:
This can be cleanup in a follow-up, but this stuff already exists in files like 
https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/HttpHeaders.html#AUTHORIZATION

Reply:
Done now using http header